.github/workflows/holochain-build-and-test.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           # we only run repo consistency checks on x86_64-linux
           - cmd:
               pkgs:
-                - build-holochain-build-crates-standalone
+                # - build-holochain-build-crates-standalone
```

Comment:
This isn't building because it's using a newer version of Rust than `hdk_derive` will build on. I'd like to temporarily disable it until we figure out running on 1.71.1

Reply:
this will be resolved by mergind and backporting https://github.com/holochain/holochain/pull/2720